plugins/node/opentelemetry-instrumentation-cassandra/src/instrumentation.ts

```diff
@@ -45,6 +45,14 @@ export class CassandraDriverInstrumentation extends InstrumentationBase {
     super('@opentelemetry/instrumentation-cassandra-driver', VERSION, config);
   }
 
+  private _getConfig(): CassandraDriverInstrumentationConfig {
+    return this._config;
+  }
+
+  override setConfig(config?: CassandraDriverInstrumentationConfig): void {
+    super.setConfig(config);
+  }
+
```

Comment:
Why this is needed?

Reply:
it's not really needed, but it gives the user another option to configure the instrumentation
eg: new CassandraDriverInstrumentation().setConfig({ })

Reply:
Oh cool